app/views/content/steps-to-become-a-teacher/_step_5_apply_for_teacher_training.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>You can usually start applying in October the year before your course starts. As courses fill up quickly, it’s important to start thinking about your course before then.</p>
+
+<p><a href="https://www.gov.uk/apply-for-teacher-training">Apply for teacher training</a>.</p>
```

Comment:
Hmm, I'm not sure. Appreciate that it's good to link people to the guidance so that they can write better applications, but then are we adding an unnecessary link when we've done quite well at cutting down the links on this page. Thoughts @philspray ?

Reply:
I think we can add an extra link to this step and signpost it as 'Get tips on making a great application including finding the right references and writing a personal statement.'